### ISSUE TITLE
Question about gradients for camera extrinsics

### ISSUE THREAD

**Issue by QT-Zhu** [facebookresearch/pytorch3d#1038]
Dear developers of PyTorch3D,

Thanks for your great work! I am recently working on a project where I attempt to optimize camera extrinsics (R & T) with differentiable rendering. I expect the sampled texels (from fragments) to be differentiable w.r.t. R & T, but it seems that the process of rasterization cannot provide gradients for R & T (`R.requires_grad=True` but `R.grad=None`). 

Does it mean, to accomplish this, I still need a shader to connect R & T into the computation graph?

Thanks in advance!
